pkg/experiment/metastore/metastore.go

```diff
@@ -178,6 +191,7 @@ func (m *Metastore) starting(context.Context) error {
 func (m *Metastore) stopping(_ error) error {
 	close(m.done)
 	m.wg.Wait()
+	m.dlq.Stop()
 	return m.Shutdown()
 }
```

Comment:
Stop is called twice: here and in `m.Shutdown`, where I believe it should live. Let's remove if from here

NB: I guess we should stop DLQ before raft just to minimize chances of concurrent recovery

Reply:
:+1: applied